opencti-platform/opencti-graphql/src/utils/filtering.js

```diff
@@ -20,14 +20,16 @@ export const ASSIGNEE_FILTER = 'assigneeTo';
 export const OBJECT_CONTAINS_FILTER = 'objectContains';
 export const RELATION_FROM = 'fromId';
 export const RELATION_TO = 'toId';
+export const INSTANCE_FILTER = 'instanceId';
```

Comment:
Why not use elementId that already exists ?

Reply:
elementId is a filter on sourceId or targetId of a relation, not on the id of the element itself

Reply:
@richard-julien It seems that we are adding multiple filters with similar naming but with different semantic. What do you think about that ?

Reply:
fixed